Unity/Assets/NativeScript/Bindings.cs

```diff
@@ -116,19 +93,14 @@ public static int GetHandle(object obj)
 				
 				lock (objects)
 				{
-					// Look up the object in the hash table
-					int initialIndex = (int)(
-						((uint)obj.GetHashCode()) % maxObjects);
-					int index = initialIndex;
-					do
+					// A handle with a value of 0 is NULL
+					int handle = 0;
```

Comment:
Because `handle` is an `out` parameter on the next line, this initialization will be overwritten and effectively does nothing. Please remove it.